pyproject.toml

```diff
@@ -17,6 +17,8 @@ dependencies = [
     "numpy",
     "trimesh",
     "zarr",
+    "cryoet-data-portal>=3.0.3",
```

Comment:
Is it actually safe to use a `>=` pin for this one? The answer can definitely be yes, but the `cryoet_data_portal` module is pretty linked to the API version now.